source/group-meeting/paper-reading-eos.rst

```diff
@@ -12,14 +12,21 @@ This website is a backup of group meeting information of `EOS Observational Seis
 - Wardah Shafiqah Binti MOHAMMAD FADIL
 - Qibin Shi
 - Hongyu Zeng
-- Win Shwe Sin OO
+- Win Shwe Sin Oo
 - Deepa
 - Yukuan Chen
 
 
 2021
 ----
 
+Fault damage zones
++++++++++++++++++++
+
+- 2019-04-28 by Win Shwee Sin Oo
+- Kim, Y. S., Peacock, D. C., & Sanderson, D. J. (2004). Fault damage zones. Journal of Structural Geology, 26(3), 503–517. 
+https://doi.org/10.1016/j.jsg.2003.08.002
```

Comment:
See https://github.com/MIGG-NTU/MIG_Docs/pull/90/checks?check_run_id=2454870513:
> /home/runner/work/MIG_Docs/MIG_Docs/source/group-meeting/paper-reading-eos.rst:28: WARNING: Bullet list ends without a blank line; unexpected unindent.

The listed term should have the same indentation if there are multiple lines.

```suggestion
- Kim, Y. S., Peacock, D. C., & Sanderson, D. J. (2004). Fault damage zones. Journal of Structural Geology, 26(3), 503–517. 
  https://doi.org/10.1016/j.jsg.2003.08.002
```